<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="bluetoothBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/bluetooth.jpg</normaloff>resources/bluetooth.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>280</width>
      <height>320</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="radioBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resources/Antena-Wireless.jpg</normaloff>resources/Antena-Wireless.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>280</width>
      <height>320</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
